templates/dashboard/page/detail.html

```diff
@@ -0,0 +1,79 @@
+{% extends "dashboard/base.html" %}
+{% load i18n %}
+{% load mptt_tags %}
+{% load materializecss %}
+{% load staticfiles %}
+{% load utils %}
+
+{% block title %}{% trans "Pages" context "Pages list page title" %} -
+  {{ block.super }}{% endblock %}
+
+{% block body_class %}body-pages{% endblock %}
+
+{% block breadcrumbs %}
+  <ul class="breadcrumbs breadcrumbs--history">
+    <li>
+      <a href="{% url 'dashboard:page-list' %}" class="breadcrumb">
+        {% trans "Pages" context "Pages form breadcrumbs" %}
+      </a>
+    </li>
+    <li class="back-mobile">
+      <a href="{% url 'dashboard:page-list' %}">
+        <svg data-src="{% static 'dashboard/images/arrow-left.svg' %}" fill="#fff" width="20px" height="20px" />
+      </a>
+    </li>
+    <li>
+      <span class="breadcrumbs--ellipsed-item">
+        {{ page }}
+      </span>
+    </li>
+  </ul>
+{% endblock %}
+
+{% block menu_pages_class %}active{% endblock %}
+
+{% block content %}
+  <div class="row">
+    <div class="col s12 l9">
+      <div class="card">
+        <div class="card-content">
+            {% page_published page=page as is_published %}
+            {% if is_published %}
+              <span class="right label label-success">PUBLISHED</span>
```

Comment:
This availability summary is displayed in two places: here and in the pages list. This is very good reason to wrap it into a reusable function, in this case probably a template tag. Also, remember that these labels should be translatable.